src/components/modals/DataCustodianFormModal.js

```diff
@@ -0,0 +1,167 @@
+import { useState, useEffect } from 'react';
+import { cloneDeep, isObject, includes, isNil, isEmpty } from 'lodash/fp';
+import { h, div, label } from 'react-hyperscript-helpers';
+import { Styles } from '../../libs/theme';
+import CloseIconComponent from '../CloseIconComponent';
+import Modal from 'react-modal';
+import Creatable from 'react-select/creatable';
+import SimpleButton from '../SimpleButton';
+import { Theme } from '../../libs/theme';
+
+//NOTE: functionality is set for demo purposes on SO Console (no admin functionality)
+//As such, changes to "data" don't persist
+
+const FormFieldRow = (props) => {
+  const {
+    user,
+    dropdownOptions,
+    updateUser,
+    setUser,
+  } = props;
+
+  const nonCustodians = dropdownOptions.filter((option) => {
+    return !option.isDataCustodian;
+  });
+  const [filteredDropdown, setFilteredDropdown] = useState(nonCustodians);
+
+  //filter function for users dropdown
+  const userListFilter = ({ searchTerm, user, setUser, action }) => {
+    let filteredCopy;
+    if (isEmpty(searchTerm)) {
+      filteredCopy = filteredDropdown;
+    } else {
+      const copiedDropdown = cloneDeep(filteredDropdown);
+      filteredCopy = copiedDropdown.filter((user) => {
+        const userNameFilter = !isEmpty(user.displayName)
+          ? includes(user.displayName)(searchTerm)
+          : false;
+        const emailFilter = !isEmpty(user.email)
+          ? includes(user.email)(searchTerm)
+          : false;
+        return userNameFilter || emailFilter;
+      });
+    }
+    setFilteredDropdown(filteredCopy);
+    if (action !== 'input-blur' && action !== `menu-close`) {
+      setUser(Object.assign({}, user, { email: searchTerm }));
+    }
+  };
+
+  let template = div({ style: { marginBottom: '2%' } }, [
+    label({}, ['Users']),
+    h(Creatable, {
+      key: 'select-user',
+      isClearable: true,
+      onChange: updateUser,
+      createOptionPosition: 'first',
+      onInputChange: (input, { action }) =>
+        userListFilter({ input, user, setUser, action }),
+      getNewOptionData: (input) => {
+        return { email: input };
+      },
+      options: dropdownOptions,
+      placeholder: 'Select or type a new user email',
+      isOptionSelected: () => false, //Workaround to prevent odd react-select behavior where all dropdown options are highlighted
+      getOptionLabel: (option) =>
+        `${option.displayName || 'New User'} (${
+          option.email || 'No email provided'
+        })` || option.email,
+    }),
+  ]);
+  return div({ display: 'flex' }, [template]);
+};
+
+export default function DataCustodianFormModal(props) {
```

Comment:
Not really feedback for this PR, but just wanted to express my hopes that we can address the modal styling at some point. Will dig around and see if we have any stories in the backlog for this task.